src/components/Reports/StaffExpenseReport/BalanceCard/BalanceCard.tsx

```diff
@@ -0,0 +1,148 @@
+  ({ theme, isSelected }) => ({
+    padding: theme.spacing(1),
+    margin: 0,
+    minHeight: 48,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: isSelected ? 'center' : 'flex-start',
+  }),
+);
+
+const StyledIconBox = styled(Box)<{ iconBgColor?: string }>(
+  ({ theme, iconBgColor }) => ({
+    backgroundColor: iconBgColor || theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+    borderRadius: theme.spacing(1),
+    padding: theme.spacing(1),
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  }),
+);
+
+const StyledCard = styled(Card)<{ isSelected: boolean }>(
+  ({ theme, isSelected }) => ({
+    padding: theme.spacing(2),
+    flex: 1,
+    minWidth: 0,
+    maxWidth: 'none',
+    fontSize: '1.25rem',
+    boxShadow: isSelected ? theme.shadows[3] : theme.shadows[1],
+    transition: 'box-shadow 0.3s ease-in-out',
+  }),
+);
+
+interface BalanceCardProps {
+  fundType: string;
+  title: string;
+  icon: React.ComponentType;
+  iconBgColor?: string;
+  startingBalance: number;
+  endingBalance: number;
+  transfersIn: number;
+  transfersOut: number;
+  onClick?: (fundType: string) => void;
+  isSelected?: boolean;
+}
+
+const ScreenOnly = styled(Box)({
+  '@media print': {
+    display: 'none',
+  },
+});
+
+export const BalanceCard: React.FC<BalanceCardProps> = ({
+  fundType,
+  title,
+  icon: Icon,
+  iconBgColor,
+  startingBalance,
+  endingBalance,
+  transfersIn,
+  transfersOut,
+  onClick,
+  isSelected = false,
+}) => {
+  const { t } = useTranslation();
+  const locale = useLocale();
+
+  return (
+    <StyledCard variant="outlined" isSelected={isSelected}>
+      <Box display={'flex'} flexDirection="row" alignItems="center" gap={1}>
+        <StyledIconBox iconBgColor={iconBgColor}>
+          <Icon />
+        </StyledIconBox>
+        <Typography variant="h6">{title}</Typography>
+      </Box>
+      <Box display="flex" flexDirection="column" mt={3} mb={2}>
+        <Typography>
+          {t('Starting Balance: ')}
+          {startingBalance.toLocaleString(locale, {
+            style: 'currency',
+            currency: 'USD',
+          })}
+        </Typography>
+        <Typography>
+          {t('+ Transfers in: ')}
+
+          {transfersIn.toLocaleString(locale, {
+            style: 'currency',
+            currency: 'USD',
+          })}
+        </Typography>
+        <Typography>
+          {t('- Transfers out: ')}
+          {Math.abs(transfersOut).toLocaleString(locale, {
+            style: 'currency',
+            currency: 'USD',
+          })}
+        </Typography>
+        <Typography>
+          {t('= Ending Balance: ')}
+          {endingBalance.toLocaleString(locale, {
+            style: 'currency',
+            currency: 'USD',
+          })}
+        </Typography>
+      </Box>
+
+      <ScreenOnly>
+        <StyledCardActionArea
+          onClick={() => {
+            onClick?.(fundType);
```

Comment:
Isn't `onClick` always provided to this component? Can we make the prop required?


```suggestion
            onClick.(fundType);
```